src/devices/src/virtio/balloon/device.rs

```diff
@@ -73,6 +73,15 @@ struct BalloonStat {
 // Safe because BalloonStat only contains plain data.
 unsafe impl ByteValued for BalloonStat {}
 
+// BalloonStats holds statistics returned from the stats_queue.
+#[derive(Clone, Default, Debug, PartialEq, Serialize)]
+pub struct BalloonState {
```

Comment:
Suggest `BalloonConfig` or some other name; `BalloonState` misleads the reader to think of snapshots 😛